app/controllers/admin/configs_controller.rb

```diff
@@ -142,7 +142,7 @@ class ConfigsController < Admin::ApplicationController
         onboarding_taskcard_image
       ].freeze
 
-    VALID_URL = %r{\A(http|https)://([/|.|\w|\s|-])*.[a-z]{2,5}(:[0-9]{1,5})?(/.*)?\z}.freeze
+    VALID_URL = %r{\A(http|https)://([/|.\w\s-])*.[a-z]{2,5}(:[0-9]{1,5})?(/.*)?\z}.freeze
```

Comment:
automatically changed by Rubocop. `|` are redundant here

Reply:
For anyone wondering, they are redundant because they are used within a character class (`[]`).